data/mods/Magiclysm/recipes/cooking.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "type": "recipe",
+    "result": "purified_meat",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "spellcraft",
```

Comment:
reverse the skill required to cooking being primary and spellcraft being secondary, for both recipes.